data/presets/baking_oven/bakehouse.json

```
@@ -0,0 +1,24 @@
+{
+    "icon": "Bakehouse",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "baking_oven",
+        "cuisine": "Bakehouse"
+    },
+    "reference": {
+        "key": "cuisine"
+    },
+    "terms": [
+        "breads",
+        "croissants",
+        "muffins",
+        "tea Cakes",
+        "bagels",
+        "nibbles",
+        "cookies"
+    ],
+    "name": "Bakehouse"
```

This name would clash with the tag `building=bakehouse` (https://wiki.openstreetmap.org/wiki/Tag:building%3Dbakehouse). Maybe one of the following would be better:

```suggestion
    "name": "Baking Oven"
```

```suggestion
    "name": "Communal Oven"
```

---

Thank you so much for the suggestion implemented the same....